integration/externalbuilders/binary/bin/run

```diff
@@ -20,6 +20,7 @@ export CORE_PEER_TLS_ENABLED="true"
 export CORE_TLS_CLIENT_CERT_FILE="$ARTIFACTS/client.crt"
 export CORE_TLS_CLIENT_KEY_FILE="$ARTIFACTS/client.key"
 export CORE_PEER_TLS_ROOTCERT_FILE="$ARTIFACTS/root.crt"
+export CORE_PEER_LOCALMSPID="$(jq -r .mspid "$ARTIFACTS/chaincode.json")"
```

Comment:
I'm not opposed to adding this line here -- though it's odd to add it only in the binary external builder and not in the golang one, especially since no chaincode (as yet) consumes this variable.